SRS.md

```diff
@@ -106,3 +121,72 @@ A seller will have access to sellers functions, while a buyer will have access t
         <tr><td>Set a transaction time frame</td></tr>
 </tr>
 </table>
+
+### 2.4 OPERATING ENVIRONMENT
+
+**The operating environment for this crypto-swap system is as listed below**
+
+- distributed database
+- client/server system
+- Operating system:  Cross-platform.
+- database: MySQL
+- platform: NodeJS
+
+### 2.5 DESIGN and IMPLEMENTATION CONSTRAINTS
+
+
+
+### 2.6 ASSUMPTION DEPENDENCIES
+
+A dispute can be awarded to the client base on the competent of evidence provided by the client (as requested by the moderator)
+
+A dispute may take up to 3 working days if enough evidence is not provided by the client 
+
+A trade may delay due to the failure of a third party ( such as mtn API not responding, network problem, etc)
+
+Any act of spam or scam may lead to the deletion of a user 
+
+
+## 3. SYSTEM FEATURES
+### 3.1 DESCRIPTION and PRIORITY
+
+The crypto-swap provides a safe secured and decentralized exchange platform where anybody can buy and sell cryptocurrencies in less than no time.
+This project has a high priority because it helps sellers compare the buying rate of various buyers and choose the best.  
+
+### 3.2 STIMULUS/RESPONSE SEQUENCES
+
+
+#### FUNCTIONAL REQUIREMENTS
+
+**Other system features include:**
+
+## 4. EXTERNAL INTERFACE REQUIREMENTS
+### 4.1 USER INTERFACES
+
+- Front-end software: ViewJs.
+- Back-end software: MySQL, Nodejs, Expressjs.
+
+### 4.2 HARDWARE INTERFACES
+
```

Comment:
This section is not application to the best of my knowledge and you can simple put NA for "Not Applicable"